### ISSUE TITLE
Modx 2.3: Doesn't Show in in Package Management

### ISSUE THREAD

**Issue by hephaestoskin** [b03tz/FormSave#23]
When I try to install this on new Modx 2.3 sites it won't show when searching for it. I have upgraded a site to 2.3 with formsave installed and it works just fine.

**Comment by sonicpunk** [comment]
I can also confirm that it works fine on MODX 2.3!

**Comment by shaunmorrison** [comment]
Yes, I couldn't see it in the Package installer either. Glad to hear it seems to work though.

**Comment by bartholomej** [comment]
+1

**Comment by BearsFan** [comment]
+1

**Comment by isaacniebeling** [comment]
Still not showing up in 2.3 sites; it does seem to work on sites where it's already installed when they're upgraded.

**Comment by digitalbart** [comment]
Can this be added, so it is visible, in the Package Management screen.

**Comment by terrybarth** [comment]
Is this being addressed. I want to use it on a project, but can't find in package management.

**Comment by bartholomej** [comment]
FYI: FormSave was already integrated into FormIt itself. Just use hook `FormItSaveForm` like this https://rtfm.modx.com/extras/revo/formit/formit.hooks/formit.hooks.formitsaveform

**Comment by terrybarth** [comment]
Oh fantastic!  Thank you!
